### ISSUE TITLE
Have anyone run it on windows 7?

### ISSUE THREAD

**Issue by leelyliu** [siddontang/go-mysql-elasticsearch#47]
When i run make on win7,I got an error as following:
godep go build -o bin/go-mysql-elasticsearch ./cmd/go-mysql-elasticsearch
process_begin: CreateProcess(NULL, godep go build -o bin/go-mysql-elasticsearch
./cmd/go-mysql-elasticsearch, ...) failed.
make (e=2): 系统找不到指定的文件。
make: *** [build-elasticsearch] 错误 2

What's wrong here?

**Comment by siddontang** [comment]
Hi @leelyliu 
I have no windows platform, so I don't know how to fix it. Sorry.

**Comment by leelyliu** [comment]
Hi @siddontang
Thanks your reply anyway. I run it successfully on Win7 at last. 

Coz no experience of the GO language, I spent too much time compiling the source on windows. 
I downloaded a "make.exe" from internet, but the compile failed. Show me above error. I have to continue to find answer on internet. Maybe there are some problems of the "make.exe". So I got a "make.exe" by MinGW. And complied your program by the "make.exe". It worked!!! 

But when I run go-mysql-elasticsearch in command line on windows. The system told me the program can't be recognized. So I changed "godep go build -o bin/go-mysql-elasticsearch" to "godep go build -o bin/go-mysql-elasticsearch.exe" in MikeFile from your src. And then compiled them again. The exe worked!!!

It's great!!
